element_array_ephys/export/nwb.py

```diff
@@ -0,0 +1,153 @@
+                shank=electrode['shank'], shank_col=electrode['shank_col'], shank_row=electrode['shank_row'],
+                location=electrode_group.location)
+
+        # ---- Units ----
+        unit_query = clustering_query @ ephys.CuratedClustering.Unit
+        for unit in unit_query.fetch(as_dict=True):
+            waveform_mean = waveform_std = np.full(1, np.nan)
+            if ephys.WaveformSet & curated_clustering_key & {'unit': unit['unit']}:
+                waveform_mean = (ephys.WaveformSet.PeakWaveform
+                                 & curated_clustering_key
+                                 & {'unit': unit['unit']}).fetch1('peak_electrode_waveform')
+                # compute waveform's STD across spikes from the peak electrode
+                electrode_waveforms = (ephys.WaveformSet.Waveform & curated_clustering_key
+                                       & {'unit': unit['unit']}
+                                       & {'electrode': unit['electrode']}).fetch1('waveforms')
+                if electrode_waveforms is not None:
+                    waveform_std = np.std(electrode_waveforms, axis=0)
+
+            nwbfile.add_unit(id=unit['unit'],
+                             electrodes=np.where(nwbfile.electrodes.id.data == int(unit['electrode']))[0],
+                             electrode_group=electrode_group,
+                             sampling_rate=unit['sampling_rate'],
+                             cluster_quality_label=unit['cluster_quality_label'],
+                             spike_times=unit['spike_times'],
+                             spike_sites=unit['spike_sites'],
+                             spike_count=unit['spike_count'],
+                             spike_depths=unit['spike_depths'],
+                             waveform_mean=waveform_mean,
+                             waveform_sd=waveform_std)
+
+        # ---- Local Field Potential ----
+        if ephys.LFP.Electrode & curated_clustering_key:
+            nwb_lfp = pynwb.ecephys.LFP(name=f'probe_{electrode_config["probe"]} - LFP')
+
+            lfp_channels = (ephys.LFP.Electrode & curated_clustering_key).fetch('electrode')
+            lfp_time_stamps = (ephys.LFP & curated_clustering_key).fetch1('lfp_time_stamps')
+
+            electrode_ind, lfp_data = [], []
+            for chn in np.unique(lfp_channels):
+                lfp_data.append((ephys.LFP.Electrode
+                                 & curated_clustering_key & {'electrode': chn}).fetch1('lfp'))
+                electrode_ind.append(np.where(nwbfile.electrodes.id.data == chn)[0][0])
+
+            electrode_table_region = nwbfile.create_electrode_table_region(
+                electrode_ind, 'Electrodes table region for LFP')
+            lfp_data = np.vstack(lfp_data).T  # sample x channel (in uV)
+
+            nwb_lfp.create_electrical_series(
+                name='processed_electrical_series',
+                data=lfp_data * 1e-6,  # convert to Volts
```

Comment:
don't convert to volts, especially if the data is ints. Instead, include 1e-6 as a conversion factor.